samples/maps/demo/map-drilldown-without-async/demo.js

```diff
@@ -0,0 +1,84 @@
+(async () => {
+
+    const nordicTopology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/nordic-countries-core.topo.json'
+    ).then(response => response.json());
+
+    const noTopology = await fetch(
+        'https://code.highcharts.com/mapdata/countries/no/no-all.topo.json'
+    ).then(response => response.json());
```

Comment:
A demo with only one possible drill down is a bit limited... Especially since Norway covers most of the height of the basic map in the first place 🇳🇴 . What if we add the other Nordic countries to make it a bit more interesting?

Reply:
@TorsteinHonsi  If the goal is to have an interesting map then it could be done as a new demo because it would be a more efficient approach to load the other maps dynamically and this demo is mainly about drilldown without async. Should this sample go in both directions? More simplified through the usage of maps that are smaller for tests and with async for all the Nordic countries for the API sample. Should the full-Nordic be also one of the main demos in the "Dynamic" category? There's already the USA map showing drill down in there.